sdk/samples/iot/README.md

```diff
@@ -92,9 +92,9 @@ This section provides an overview of the different samples available to run and
 
 ### IoT Hub Plug and Play Multiple Component
 
-- *Executable:* `paho_iot_hub_pnp_component_example`
+- *Executable:* `paho_iot_hub_pnp_component_sample`
 
-This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_pnp_component_example.c) connects an IoT Plug and Play enabled device with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json). X509 self-certification is used.
+This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_pnp_component_sample.c) connects an IoT Plug and Play enabled device with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json). X509 self-certification is used.
```

Comment:
Link verification doing its job :)
https://dev.azure.com/azure-sdk/public/_build/results?buildId=487729&view=logs&j=a129effc-2dd1-54d1-fb5a-ad7bdc0e851d&t=ccad100f-2904-5913-2083-c525ad929a03&l=41

> WARNING: [404] broken link ***/blob/master/sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

Nice! cc  @KarishmaGhiya, @danieljurek 

That said, because all our links have to point to master, this is an expected developer pain-point when updating links whenever files move.

So, I think we need to stage such changes in multiple PRs. One approach could be:
1) Check-in a link that is not pointing to master.
2) Add a subsequent PR to fix up the link to point to master once the file has been renamed.

OR:
1) Don't add the link (i.e. remove it).
2) Add the link after the file has been checked-in.

Reply:
Interesting that makes sense. I'll remove to add the file and then send another PR.